src/components/one_click_address/OneClickAddress.jsx

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable no-mixed-operators */
+/* eslint-disable react/forbid-prop-types */
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { InputField, SelectField } from '@boldcommerce/stacks-ui';
+import '../address/Address.css';
+
+const OneClickAddress = ({
+  address,
+  onChange,
+  errors,
+  countries,
+  provinces,
+  showPostalCode,
+  showProvince,
+  provinceLabel,
+  submit,
+}) => {
+  const countryList = countries.map((countryItem) => <option value={countryItem.iso_code} key={countryItem.iso_code}>{countryItem.name}</option>);
+  const provinceList = provinces.map((provinceItem) => <option value={provinceItem.iso_code} key={provinceItem.iso_code}>{provinceItem.name}</option>);
+
+//   useEffect(() => {
```

Comment:
Can we remove this commented code? @jonathanzredfern

Reply:
Yup, good catch